### PR TITLE
move test from integration to functional

### PR DESCRIPTION
**Pull request by kdaily**
*Issue #, if available:*

*Description of changes:*

Test should be in functional, not integration.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by kyleknap — COMMENTED** [review]
Just had a small comment. Looks fine otherwise

File: tests/integration/botocore/test_sts.py [hunk 1]
```diff
@@ -40,29 +40,3 @@ def test_regionalized_endpoints(self):
...
-        'sts', region_name=None, endpoint_url="https://custom.endpoint.aws"
-    )
-    assert client.meta.region_name is None
-
-    mock_response_body = b"""\
-<AssumeRoleWithSAMLResponse xmlns="https://sts.amazonaws.com/doc/2011-06-15/">
-    <AssumeRoleWithSAMLResult></AssumeRoleWithSAMLResult>
-</AssumeRoleWithSAMLResponse>
-"""
-    with ClientHTTPStubber(client) as http_stubber:
```
Comment (kyleknap) [inline tests/integration/botocore/test_sts.py:60]: We should remove the added import of `ClientHTTPStubber`. It got added as part of the original port: https://github.com/aws/aws-cli/pull/7465

Comment (kdaily) [reply tests/integration/botocore/test_sts.py:60]: Removed it, thanks!

**Review by kyleknap — APPROVED** [review]
Thanks! :ship:

**Pull request merged**